source/PlayState.hx

```diff
@@ -168,13 +173,19 @@ class PlayState extends FlxState
 		FlxG.overlap(player, dtmEnemy1, dtmEnemy1.playerHitResolve);
 		FlxG.overlap(player, bullets, bulletHitPlayer);
 		FlxG.overlap(player, trap._grpBarTrap, trap.playerTrapResolve);
+		FlxG.overlap(player, sword, player.pickup_item);
+		FlxG.overlap(sword.hitbox.hitboxFrames, dtmEnemy1, sword.hit_enemy);
 		
 		// Add collision logic
 		FlxG.collide(blockGroup, player);
 		FlxG.collide(player, sentry1);
 		FlxG.collide(map, dtmEnemy1);
 		FlxG.collide(map, bullets);
 		FlxG.collide(blockGroup, bullets);
+		if(player.equipped_item != sword){
+			FlxG.collide(map, sword);
+			FlxG.collide(blockGroup, sword);
```

Comment:
These will have to be rewritten in the event that any other items are introduced or there are multiple swords in a level.  It might be better to put all items in a group and run collisions on that.  If there are issues with collisions when an item is being carried, it could just be pulled out of the group when picked up and added back when dropped.